projects/packages/forms/src/contact-form/class-contact-form-plugin.php

```diff
@@ -1179,18 +1179,34 @@ public function get_post_content_for_csv_export( $post_id ) {
 	 * @return mixed
 	 */
 	public function get_post_meta_for_csv_export( $post_id ) {
-		$md                                        = get_post_meta( $post_id, '_feedback_extra_fields', true );
-		$md[ __( 'Date', 'jetpack-forms' ) ]       = get_the_date( DATE_RFC3339, $post_id );
-		$content_fields                            = self::parse_fields_from_content( $post_id );
-		$md[ __( 'IP Address', 'jetpack-forms' ) ] = ( isset( $content_fields['_feedback_ip'] ) ) ? $content_fields['_feedback_ip'] : 0;
+		$md                     = get_post_meta( $post_id, '_feedback_extra_fields', true );
+		$md['-3_response_date'] = get_the_date( 'Y-m-d H:i:s', $post_id );
```

Comment:
Hmm, do you think it'd help if we had a helper method for generating these keys, to make it easier to understand what's going on?  
I'm fine with it if we decide to address it separately, but let's create an issue for it and assign that to the forms project so we can keep track of it.

Reply:
Yes, it "might" make sense. But it would also complicate things where things are already quite complex. This all needs some partial refactors here and there, field/column order just _works_ and it's based on a _de facto_ criteria.

So, for now, I'd say those keys are enough self-explanatory for when we address this more elegantly. No need for an issue, it will all blow up with #28723 and likely split it from there.

Reply:
While I wouldn't call them self-explanatory if you just happen to see the keys without the method and the comment explaining them, the explanation is there and I agree it's not worth diving too deep into it yet when we're planning to do some broader cleanup. So all good